src/FileFormats/LP/LP.jl

```diff
@@ -4,6 +4,12 @@ import ..FileFormats
 import MathOptInterface
 const MOI = MathOptInterface
 
+if isdefined(Base, :Grisu)
+    import Base.Grisu
+else
+    import Grisu
+end
```

Comment:
What's the advantage of using the `Base` version over just this package?

Reply:
It avoids having to load the package, which is a bit faster. I think it loads pretty quickly though so perhaps doesn't matter.